docs/mobile-apps/features/audio-capture.md

```diff
@@ -0,0 +1,81 @@
+---
+id: audio-capture
+title: Audio Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Audio Capture 
+
+Audio Capture is a functionality that gives you the ability to record the audio stream generated by your native mobile app (Android or iOS/iPadOS) during a live or automated real device test. 
+
+With this feature, the audio is recorded on our side and merged with the video recording, which you can access in your **Test Results**.
+
+## What You'll Need
+
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+* A native Android, iOS, or iPadOS mobile app that makes HTTP/HTTPS requests. We currently support the following:
+  * Android 10 and higher
+  * iOS 10+ up to iOS 15.3
+
+
+## Using Audio Capture on Real Devices
+
+To enable audio capture in your tests: 
+ 
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="800"/>
+4. Under Default Settings, toggle Instrumentation and App/OS Audio to enable the feature. <br/><img src={useBaseUrl('img/mobile-apps/audio_capture_screen2.png')} alt="Mobile app settings navigation" width="800"/>
+5. **For Automated Testing only:** add the `audioCapture` capability to your test script. Click the link below corresponding to your framework:
+   * Espresso (coming soon)
+   * XCUITest (coming soon)
```

Comment:
@aoleszyn Once [this](https://github.com/saucelabs/sauce-docs/pull/1161/files#r850751762) is done, please finish adding the links.

Reply:
lets do that in an other PR as we did with network capture. (feature went out without Espresso/Xcuit) backend not yet ready